### PR TITLE
Added pricing & context info for deepseek chat (v2) and deepseek coder

### PR DESCRIPTION
**Pull request by paul-gauthier**

### CODE REVIEW

File: model_prices_and_context_window.json [hunk 1]
```diff
@@ -739,6 +739,24 @@
         "litellm_provider": "mistral",
         "mode": "embedding"
     },
+    "deepseek-chat": {
+        "max_tokens": 4096,
+        "max_input_tokens": 32000,
+        "max_output_tokens": 4096,
+        "input_cost_per_token": 0.00000014,
+        "output_cost_per_token": 0.00000028,
+        "litellm_provider": "openai",
```
Comment (ishaan-jaff) [inline model_prices_and_context_window.json:748]: openai API provides `deepseek-chat` ?

**Comment by paul-gauthier** [comment]
In the current litellm, you can only access deepseek's apis via `OPENAI_API_BASE` and `OPENAI_API_KEY`.

I have a PR coming to add first class support for deepseek's api.

**Comment by paul-gauthier** [comment]
PR #3503 may supersede this one if it looks acceptable to you.

**Comment by ishaan-jaff** [comment]
can we close this  in favor of : https://github.com/BerriAI/litellm/pull/3503

**Pull request closed without merging**